src/uu/env/src/env.rs

```diff
@@ -308,7 +314,41 @@ fn run_env(args: impl uucore::Args) -> UResult<()> {
          * created. This is much simpler than dealing with the hassles of calling execvp directly.
          */
         match process::Command::new(&*prog).args(args).status() {
-            Ok(exit) if !exit.success() => return Err(exit.code().unwrap().into()),
+            Ok(exit) if !exit.success() => {
+                #[cfg(unix)]
+                if let Some(exit_code) = exit.code() {
+                    return Err(exit_code.into());
+                } else {
+                    // `exit.code()` returns `None` on Unix when the process is terminated by a signal.
+                    // See std::os::unix::process::ExitStatusExt for more information.
+                    let signal_code = exit.signal().unwrap();
+                    println!(
+                        "{}{}",
+                        unsafe {
```

Comment:
Maybe https://github.com/uutils/coreutils/blob/5eeac5881a250feb1a289d69b6d0d05b33a6d8d5/src/uucore/src/lib/features/signals.rs#L98 works here?

Reply:
It does, thanks! When I was looking into how to test, I came across it and I'm using that now.

However, I'm still not sure how to test, considering what I did before was run a program that raised a signal - I don't know how to raise a signal straight from the command line.

Reply:
`kill` will send a signal to a process.